x-pack/filebeat/module/zoom/webhook/_meta/fields.yml

```diff
@@ -0,0 +1,611 @@
+        User type related to the user event
+    - name: user.phone_number
+      type: keyword
+      description: >
+        Phone number related to the user
+    - name: user.phone_country
+      type: keyword
+      description: >
+        Country code related to the phone number
+    - name: user.company
+      type: keyword
+      description: >
+        User Company
+    - name: user.pmi
+      type: keyword
+      description: >
+        User personal meeting ID
+    - name: user.use_pmi
+      type: boolean
+      description: >
+        If use PMI is enabled
+    - name: user.pic_url
+      type: keyword
+      description: >
+        URL to the profile picture
+    - name: user.vanity_name
+      type: keyword
+      description: >
+        Name of the related users personal meeting room
+    - name: user.timezone
+      type: keyword
+      description: >
+        Timezone configured for the user
+    - name: user.language
+      type: keyword
+      description: >
+        Language settings for the user
+    - name: user.host_key
+      type: keyword
+      description: >
+        Host key set for the user
+    - name: user.role
+      type: keyword
+      description: >
+        The configured role for the user
+    - name: user.dept
+      type: keyword
+      description: >
+        The configured departement for the user
+    - name: user.presence_status
+      type: keyword
+      description: >
+        Current presence status of user
+    - name: user.personal_notes
+      type: keyword
+      description: >
+        Personal notes for the User
+    - name: user.date_time
+      type: date
+      description: >
+        Time when user logged in/ou
+    - name: webinar.id
+      type: keyword
+      description: >
+        Unique ID for the webinar
+    - name: webinar.join_url
+      type: keyword
+      description: >
+        Join URL for the webinar
+    - name: webinar.uuid
+      type: keyword
+      description: >
+        UUID for the webinar
+    - name: webinar.host_id
+      type: keyword
+      description: >
+        UserID of the host of the webinar
+    - name: webinar.topic
+      type: keyword
+      description: >
+        Topic of the webinar
+    - name: webinar.type
+      type: keyword
+      description: >
+        Type of webinar created
+    - name: webinar.start_time
+      type: date
+      description: >
+        Start time of the webinar
+    - name: webinar.timezone
+      type: keyword
+      description: >
+        Timezone of the webinar
+    - name: webinar.duration
+      type: long
+      description: >
+        Duration of the webinar
+    - name: webinar.agenda
+      type: keyword
+      description: >
+        Agenda of the webinar
+    - name: webinar.password
+      type: keyword
+      description: >
+        Password for the webinar
+    - name: webinar.issues
+      type: keyword
+      description: >
+        Related issues to the webina
+    - name: zoomroom.id
```

Comment:
are we grabbing these namespaces from official webhook docs? just wondering because `zoom.zoomroom.*` seems silly 😅